contracts/implementations/nodes/SyntheticDevice.sol

```diff
@@ -82,6 +91,79 @@ contract SyntheticDevice is AccessControlInternal {
 
     // ***** Interaction with nodes *****//
 
+    /**
+     * @notice Mints a list of synthetic devices and links them to vehicles
+     * To be called for existing vehicles already connected
+     * @dev Caller must have the admin role
+     * @dev All devices will be minted under the same integration node
+     * @param data input data with the following fields:
+     *  vehicleNode -> Vehicle node id
+     *  syntheticDeviceAddr -> Address associated with the synthetic device
+     *  attrInfoPairs -> List of attribute-info pairs to be added
+     */
+    function mintSyntheticDeviceBatch(
```

Comment:
Hmm let's merge in dev. Is this `@param` still complete?

Reply:
Ops